Bottom: fd0012f2d2114978488547b6e7c14d49c1ebb804
Top:    25bb31fab17bb8be37f7d3d59b6bc85276722909
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 17:50:37 +0100

hid/gtk: Replace deprecated function call gtk_box_pack_start_defaults()


---

diff --git a/src/hid/gtk/gui-dialog.c b/src/hid/gtk/gui-dialog.c
index 93cea47..1770bba 100644
--- a/src/hid/gtk/gui-dialog.c
+++ b/src/hid/gtk/gui-dialog.c
@@ -76,7 +76,7 @@ ghid_dialog_input (const char * prompt, const char * initial)
     gtk_entry_set_text (GTK_ENTRY (entry), initial);
 
   gtk_entry_set_activates_default (GTK_ENTRY (entry), TRUE);
-  gtk_box_pack_start_defaults (GTK_BOX (vbox), entry);
+  gtk_box_pack_start (GTK_BOX (vbox), entry, TRUE, TRUE, 0);
 
   content_area = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
   gtk_container_add (GTK_CONTAINER (content_area), vbox);
